datastore/tests/unit/test_transaction.py

```diff
@@ -12,22 +12,55 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import mock
```

Comment:
This is not in the standard library, it should be in another imports section:

```python
import unittest

import mock
```

FWIW `unittest.mock` **is** in the standard library, but only in Python 3.